app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -133,7 +108,10 @@ export class MergeConflictsDialog extends React.Component<
    */
   private onCancel = async () => {
     const anyResolvedFiles =
-      getResolvedFiles(this.props.workingDirectory).length > 0
+      getResolvedFiles(
```

Comment:
Extracting this bit to a named constant would help make the intent of this call clearer to me.